README.rst

```diff
@@ -90,7 +90,64 @@ By default, blockstore is run as an app inside of Open edX. Enable it using the
 If you need to run blockstore as a separate service (e.g. for scalability or performance reasons), you can deploy blockstore in production using `the blockstore ansible role <https://github.com/openedx/configuration/tree/master/playbooks/roles/blockstore>`_.
 
 -------------------------------------------------------
-Running and testing as a separate service (development)
+Development in Devstack
+-------------------------------------------------------
+
+If you want to develop blockstore as a local python package installed in edx-platform, use the following steps:
+
+#. Prerequisite: Have an Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed and working. Your devstack must use the Nutmeg release of Open edX (or newer) or be tracking the ``master`` branch of ``edx-platform``.
+
+#. To run blockstore as an app inside of Open edX, enable it using the waffle switch `blockstore.use_blockstore_app_api <https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html#featuretoggle-blockstore.use_blockstore_app_api>`_.
+
```

Comment:
I'm assuming it doesn't matter whether this is done in LMS admin or Studio admin, but might be best to be clear on the point.

Reply:
The admin waffle switch form allows you to activate a switch, not enable it. Might be best to use 'activate' as the verb here.

Reply:
> There's a lint commit error that needs attention

I'll just squash and merge.

Reply:
The link's text inappropriately includes the phrase "in LMS or Studio Admin", making it hard to see that the clarification isn't just part of the switch name.